content/docs/protoc-installation.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Protocol Buffer Compiler Installation
+short: Protoc Installation
+description: How to install the protocol buffer compiler.
+protoc-version: 3.11.4
+---
+
+While not mandatory, gRPC applications often leverage [Protocol Buffers][pb] for
+service definitions and data serialization. Most of the example code from this
+site uses [version 3 of the protocol buffer language (proto3)][proto3].
+
+The protocol buffer compiler, `protoc`, is used to compile `.proto` files, which
+contain service and message definitions. Choose one of the methods given below
+to install `protoc`.
+
+### Install using a package manager
+
+You can install the protocol compiler, `protoc`, with a package manager under
+Linux or macOS using the following commands.
```

Comment:
Some notes on the linux installation:
we should ad a note that  while installing the linux package `protobuf-compiler` is generally the easiest way, it won't give you the latest released version, so you might not get access to the latest fixes and features (which might or might not be a problem, depending on your use).

Note that to be able to use protoc with grpc, besides `protoc` you'll also need the grpc-specific plugins that generate the grpc service stubs/clients. I'm not sure how useful it is to just obtain `protoc` (there seems to be protobuf-compiler-grpc package, but I'm afraid it's too old).

Current package versions in major distros:
Current debian stable is at 3.6 (which is OKish, but not great)
https://packages.debian.org/buster/protobuf-compiler
Current ubuntu stable is at 3.0.0 which is borderline (several years old).
https://packages.ubuntu.com/bionic/protobuf-compiler

A few more notes:
- The apt command only exists on debian-based distros.
- on mac brew  will give you a much newer version than apt / apt-get 
- The note "Ensure compiler version is 3+" makes sense, because anything before 3.0.0 is definitely too old
- in some languages, we provide a "tools" package that has a pre-compiled version of protoc and the corresponding grpc plugin bundled (e.g. grpcio.tools for python or Grpc.Tools for C#).